roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/entity/CarTest.java

```diff
@@ -126,4 +126,34 @@ public void getNextSection_currentSectionNotEqualDestination() {
         assertEquals(car.getCurrentSection(), currentSectionMock);
         assertNotEquals(car.getCurrentSection(), destinationMock);
     }
+
+    @Test
+    public void getTimeToTraverseCurrentSection_isExpectedTime() {
+        RoundaboutSimulationModel model = getPrepareModel();
+        IStreetSection section = new StreetSection(10.0, null, null, model, "", false);
+        IDriverBehaviour driverBehaviour = new DriverBehaviour(5.0, 2.0, 2.0, 1.5);
+
+        IRoute route = new Route();
+        route.addSection(section);
+
+        ICar car = new Car(2, driverBehaviour, route, model, "", false);
+
+        Assert.assertEquals(2.0, car.getTimeToTraverseCurrentSection(), 0.0);
+    }
+
+    @Test
+    public void getTimeToTraverseSection_carIsNotOnThisSection(){
+        RoundaboutSimulationModel model = getPrepareModel();
+        IStreetSection currentSection = new StreetSection(10.0, null, null, model, "", false);
+        IStreetSection otherSection = new StreetSection(20.0, null, null, model, "", false);
+        IDriverBehaviour driverBehaviour = new DriverBehaviour(5.0, 2.0, 2.0, 1.5);
+
+        IRoute route = new Route();
+        route.addSection(currentSection);
+
+        ICar car = new Car(2, driverBehaviour, route, model, "", false);
+
+        Assert.assertEquals(4, car.getTimeToTraverseSection(otherSection), 0.0);
```

Comment:
if car is not in this section there should not be a valid value returned (see above)